README.md

```diff
@@ -155,34 +155,37 @@ public function main() {
 
 ### SAP Consumer
 
+The following example demonstrates how to receive an IDoc from a SAP instance.
+
 ```ballerina
 import wso2/sap;
 import ballerina/io;
 
 listener sap:Listener consumerEP = new ({
-    transportName:"<The protocol name[idoc/bapi]>",
-    serverName:"<Name of the server configuration>",
-    gwHost:"<Gateway host on which the server should be registered>",
-    progId:"<The program ID with which the registration is done>",
-    repositoryDestination:"<Name of the .dest file>",
-    gwServ:"<Gateway service>",
-    unicode:"<Determines whether or not you connect in unicodemode>"}, {
-    sapClient:"<SAP client, for example, 001>",
-    userName:"<The user logon>",
-    password:"<The logon password>",
-    asHost:"<The R/3 application server>",
-    sysnr:"<SAP system number, for example, 01>",
-    language:"<The logon language>"
+    transportName: "<The protocol name[idoc/bapi]>",
+    serverName: "<Name of the server configuration>",
+    gwhost: "<Gateway host on which the server should be registered>",
+    progid: "<The program ID with which the registration is done>",
+    repositorydestination: "<Name of the repository>",
+    gwserv: "<Gateway service>",
+    unicode: "<Determines whether or not you connect in unicodemode>"}, 
+    {
+    ^"client: "<SAP client, for example, 001>",
+    username: "<The user logon>",
+    password: "<The logon password>",
+    ashost: "<The R/3 application server>",
+    sysnr: "<SAP system number, for example, 01>",
+    language: "<The logon language>"
 });
 service SapConsumerTest on consumerEP {
     // The `resource` registered to receive server messages
-    resource function onMessage(string idoc) {
-        io:println("The message received from SAP instance : " + idoc);
+    resource function onMessage(xml idoc) {
+       io:println("Message received from SAP instance: ", idoc);
```

Comment:
Can we use log instead of io in these examples? We usually use log in services and io in main functions.